### PR TITLE
Update README.md with more Viridian details

### PR DESCRIPTION
**Pull request by EugeneAbramchuk**
Add missing SSL configuration properties and provided samples

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -226,7 +233,9 @@ The following list contains the properties supported by the `Hazelcast JDBC Driv
 For connecting to the Hazelcast cloud you only need to specify `discoveryToken` property and use the `cluster-name` as a host in
 the URL: `jdbc:hazelcast://<cluster-name>/?discoveryToken=<yourDiscoveryToken>`.
 
-If you're connecting to [Hazelcast Viridian](https://viridian.hazelcast.com/), also specify `cloudUrl=https://api.viridian.hazelcast.com`. 
+If you're connecting to [Hazelcast Viridian](https://viridian.hazelcast.com/), also specify `cloudUrl=https://api.viridian.hazelcast.com` and `sslEnabled=true`: `jdbc:hazelcast://<cluster-name>/?discoveryToken=<yourDiscoveryToken>&cloudUrl=https://api.viridian.hazelcast.com&sslEnabled=true`.
```
Comment (angelasimms) [inline README.md:236]:
```suggestion
If you're connecting to a [Viridian Serverless cluster](https://viridian.hazelcast.com/), also specify `cloudUrl=https://api.viridian.hazelcast.com` and `sslEnabled=true`: `jdbc:hazelcast://<cluster-name>/?discoveryToken=<yourDiscoveryToken>&cloudUrl=https://api.viridian.hazelcast.com&sslEnabled=true`. 
```

File: README.md [hunk 2]
```diff
@@ -226,7 +233,9 @@ The following list contains the properties supported by the `Hazelcast JDBC Driv
 For connecting to the Hazelcast cloud you only need to specify `discoveryToken` property and use the `cluster-name` as a host in
 the URL: `jdbc:hazelcast://<cluster-name>/?discoveryToken=<yourDiscoveryToken>`.
 
-If you're connecting to [Hazelcast Viridian](https://viridian.hazelcast.com/), also specify `cloudUrl=https://api.viridian.hazelcast.com`. 
+If you're connecting to [Hazelcast Viridian](https://viridian.hazelcast.com/), also specify `cloudUrl=https://api.viridian.hazelcast.com` and `sslEnabled=true`: `jdbc:hazelcast://<cluster-name>/?discoveryToken=<yourDiscoveryToken>&cloudUrl=https://api.viridian.hazelcast.com&sslEnabled=true`. 
+
+TLS is mandatory for Viridian cluster connections so pass TLS properties such as truststore, keystore etc along to `DriverManager.getConnection()`.
```
Comment (angelasimms) [inline README.md:238]:
```suggestion
TLS is mandatory for Viridian Serverless cluster connections so pass TLS properties such as truststore, keystore etc along to `DriverManager.getConnection()`.
```

**Review by angelasimms — CHANGES_REQUESTED** [review]

**Review by angelasimms — APPROVED** [review]
LGTM

File: README.md [hunk 3]
```diff
@@ -226,7 +233,9 @@ The following list contains the properties supported by the `Hazelcast JDBC Driv
 For connecting to the Hazelcast cloud you only need to specify `discoveryToken` property and use the `cluster-name` as a host in
 the URL: `jdbc:hazelcast://<cluster-name>/?discoveryToken=<yourDiscoveryToken>`.
 
-If you're connecting to [Hazelcast Viridian](https://viridian.hazelcast.com/), also specify `cloudUrl=https://api.viridian.hazelcast.com`. 
+If you're connecting to a [Viridian Serverless cluster](https://viridian.hazelcast.com/), also specify `cloudUrl=https://api.viridian.hazelcast.com` and `sslEnabled=true`: `jdbc:hazelcast://<cluster-name>/?discoveryToken=<yourDiscoveryToken>&cloudUrl=https://api.viridian.hazelcast.com&sslEnabled=true`. 
+
+TLS is mandatory for Viridian Serverless cluster connections so pass TLS properties such as truststore, keystore etc along to `DriverManager.getConnection()`.
```
Comment (viliam-durina) [inline README.md:238]: I find it confusing that we use SSL and TLS interchangeably. For example, "You should use TLS, therefore specify `sslEnabled`".

```suggestion
SSL is mandatory for Viridian Serverless cluster connections so add SSL properties such as `trustStore`, `keyStore` etc. in the URL.
```

File: README.md [hunk 4]
```diff
@@ -226,7 +233,9 @@ The following list contains the properties supported by the `Hazelcast JDBC Driv
 For connecting to the Hazelcast cloud you only need to specify `discoveryToken` property and use the `cluster-name` as a host in
 the URL: `jdbc:hazelcast://<cluster-name>/?discoveryToken=<yourDiscoveryToken>`.
 
-If you're connecting to [Hazelcast Viridian](https://viridian.hazelcast.com/), also specify `cloudUrl=https://api.viridian.hazelcast.com`. 
+If you're connecting to a [Viridian Serverless cluster](https://viridian.hazelcast.com/), also specify `cloudUrl=https://api.viridian.hazelcast.com` and `sslEnabled=true`: `jdbc:hazelcast://<cluster-name>/?discoveryToken=<yourDiscoveryToken>&cloudUrl=https://api.viridian.hazelcast.com&sslEnabled=true`.
```
Comment (viliam-durina) [inline README.md:236]: Would be better to refer to the URL above.

File: README.md (hunk 3 above)
Comment (EugeneAbramchuk) [reply README.md:238]: I think we do not support SSL now at all, so TLS is a better name. The query parameter actually seems to be a misnomer which should be renamed to `tlsEnabled`.

File: README.md [hunk 5]
```diff
@@ -128,7 +128,14 @@ The URL must have the following structure, parts in `[]` are optional:
...
+To connect to a Viridian Serverless cluster, use:
+```
+jdbc:hazelcast://<cluster-name>/?discoveryToken=<yourDiscoveryToken>&cloudUrl=https://api.viridian.hazelcast.com&sslEnabled=true`
+```
+Note: Viridian Serverless clusters require [additional client configuration for TLS](https://github.com/hazelcast/hazelcast-jdbc#hazelcast-cloud-configuration).
+
+
+To connect to the legacy Cloud service, use:
 ```
 jdbc:hazelcast://cluster-id/discoveryToken=value[&property1=value1[&property2=value2]...]
```
[the remark this answers is not in the document]
Comment (EugeneAbramchuk) [reply README.md:140]: It is closer to cluster id indeed, and has a `pr-` prefix. I updated all details correspondingly.

**Review by k-jamroz — APPROVED** [review]

**Pull request merged**